hydra-test-utils/src/Test/Hydra/Prelude.hs

```diff
@@ -133,16 +134,22 @@ combinedHspecFormatter suiteName config = do
 --
 -- @@
 -- withCreateProcess p $
---   \_stdin _stdout _stderr processHandle -> do
+--   \_stdin _stdout stderr processHandle -> do
 --       race_
---         (checkProcessHasNotDied "my-process" processHandle)
+--         (checkProcessHasNotDied "my-process" processHandle stderr)
```

Comment:
Must: fix this example.

I think this does not compile as the `withCreateProcess` gives a `Maybe Handle`. We could consider changing the `checkProcessHasNotDied` to take also a `Maybe Handle` and only print if it was `Just`.

Please also mention in the haddock that `Inherit` handles will **NOT** work.